front/src/applications/operationalStudies/components/ManageTrainSchedule/Itinerary/DisplayItinerary/RunningTime.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { ComponentType, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { updateMaximumRunTime } from 'reducers/osrdconf';
+import InputSNCF from 'common/BootstrapSNCF/InputSNCF';
+import { Dispatch } from 'redux';
+import { noop } from 'lodash';
+import { RxLapTimer } from 'react-icons/rx';
+import 'styles/main.css';
+
+interface RunningTimeProps {
+  dispatch?: Dispatch;
+}
+
+export function withProps<T>(Component: ComponentType<T>) {
+  return function runTimeWithProps(hocProps: T) {
+    const dispatch = useDispatch();
+    return <Component {...(hocProps as T)} dispatch={dispatch} />;
+  };
+}
+
+function RunningTime({ dispatch = noop }: RunningTimeProps) {
+  const [maximumRuntime, setMaximumRunTime] = useState<string | undefined>(undefined);
+  const [isRtBelowCap, setIsRtBelowCap] = useState(true);
+  const runTimeCap = 43200;
+
+  function getSeconds(time: string): number | undefined {
```

Comment:
- Function's name is not clear : get seconds from what ? to do what ? I understand this is a way to test if input value is under or equal to max, but it's not specified.
- `time2sec` & `sec2time` already exists, you should use them
- you compare strings with `>` ? I suggest to compare seconds and not strings, you convert time in seconds and compare them (using previously defined constant `runTimeCap`